recycle.xml

```diff
@@ -0,0 +1,210 @@
+<!-- 1 Enrichment 2 Reactors -->
```

Comment:
this file needs an updated comment block and should be moved either to `cycamore/input` or more preferably `cycamore/tests/input` with an associated regression test